src/doc/using_visit/MakingItPretty/Color_tables.rst

```diff
@@ -523,16 +520,20 @@ Tags must now only be comprised of alphanumeric characters, whitespace, dashes,
 Searching for Color Tables
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Another option for finding the right color table is to enable searching. 
-With this option, users can type a search term into the **Name** text box, and only color tables that have a name containing that term will appear in the color table name table.
+Another option for finding the right color table is to use the search bar.
+With this option, users can type a search term into the **Search** text box, and only color tables that have a name containing that term will appear in the color table name table.
+Prior to VisIt_ 3.4, the **Name** text box doubled as the search bar, and users had to check the box for it to the right of the **Name** text box.
```

Comment:
Don't think this is sentance is needed.

Reply:
Ok. I was worried about having documentation that does not reflect the VisIt that users are using, but I suppose they can look at earlier versions of the docs if they want to know what is true for their version of visit.